packages/invoice-dashboard/src/lib/view-requests.svelte

```diff
@@ -152,7 +152,13 @@
     if (typeof unwatchAccount === "function") unwatchAccount();
   });
 
-  $: cipherProvider = undefined;
+  $: cipherProvider =
```

Comment:
I accidentally pushed the undefined value while developing here as well

Reply:
I wondered about these but since it happened in two places, I assumed it was a refactor.. I should have looked closer and asked about it.